packages/nx/src/schematics/application/application.ts

```diff
@@ -1,75 +1,37 @@
-import {
-  apply,
-  chain,
-  Tree,
-  Rule,
-  url,
-  move,
-  template,
-  mergeWith,
-  branchAndMerge,
-  SchematicContext,
-  SchematicsException,
-  schematic,
-  noop,
-  externalSchematic,
-} from '@angular-devkit/schematics';
-import {
-  prerun,
-  missingArgument,
-  PluginHelpers,
-  updatePackageScripts,
-  getPrefix,
-  updateWorkspace,
-  updateNxProjects,
-  getAppName,
-  getDefaultTemplateOptions,
-  getFrontendFramework,
-} from '../../utils';
-import { nsWebpackVersion } from '../../utils/versions';
+import { apply, branchAndMerge, chain, externalSchematic, mergeWith, move, noop, Rule, SchematicContext, SchematicsException, template, Tree, url } from '@angular-devkit/schematics';
+import { addDepsToPackageJson } from '@nrwl/workspace';
+import { getAppName, getDefaultTemplateOptions, getFrontendFramework, getPrefix, missingArgument, PluginHelpers, prerun, updateNxProjects, updatePackageScripts, updateWorkspace } from '../../utils';
+import { nsAngularVersion, nsRxjs, nsWebpackVersion, nsZonejs } from '../../utils/versions';
 import { Schema } from './schema';
 
 export default function (options: Schema) {
   if (!options.name) {
-    throw new SchematicsException(
-      missingArgument(
-        'name',
-        'Provide a name for your NativeScript app.',
-        'nx g @nativescript/nx:app name'
-      )
-    );
+    throw new SchematicsException(missingArgument('name', 'Provide a name for your NativeScript app.', 'nx g @nativescript/nx:app name'));
   }
 
   return chain([
     prerun(options, true),
     PluginHelpers.applyAppNamingConvention(options, 'nativescript'),
-    (tree: Tree, context: SchematicContext) =>
-      addAppFiles(options, options.name),
+    (tree: Tree, context: SchematicContext) => addAppFiles(options, options.name),
     // add extra files per options
-    (tree: Tree, context: SchematicContext) =>
-      options.routing && ['angular'].includes(options.framework)
-        ? addAppFiles(options, options.name, 'routing')
-        : noop(),
+    (tree: Tree, context: SchematicContext) => (options.routing && ['angular'].includes(options.framework) ? addAppFiles(options, options.name, 'routing') : noop()),
     // add app resources
     (tree: Tree, context: SchematicContext) =>
       externalSchematic(
         '@nativescript/nx',
         'app-resources',
         {
-          path: `apps/${options.directory ? options.directory + '/' : ''}${
-            options.name
-          }`,
+          path: `apps/${options.directory ? options.directory + '/' : ''}${options.name}`,
         },
         { interactive: false }
       )(tree, context),
+    addAngularDependencies(),
```

Comment:
Let's just do this here so it doesn't assume angular all the time:
```
// add frontend framework dependencies
(tree: Tree, context: SchematicContext) => (['angular'].includes(options.framework) ? addAngularDependencies() : noop()),
```

Reply:
Good catch! Done https://github.com/NativeScript/nx/pull/1/commits/868aab5c7e11a900ea32b175cb44dddbe76d6664